### PR TITLE
small secret doc update

### PR DESCRIPTION
**Pull request by vieux**
ping @mstanleyjones

### CODE REVIEW

File: docs/reference/commandline/secret_create.md [hunk 1]
```diff
@@ -34,32 +34,32 @@ command on a manager node.
 ### Create a secret
 
 ```bash
-$ cat secret.json | docker secret create -f - secret.json
+$ echo <secret> | docker secret create -f - my_secret
```
Comment (mstanleyjones) [inline docs/reference/commandline/secret_create.md]: Is the `-f -` required? I think that's a really strange syntax. Do any other Docker commands work like that?

File: docs/reference/commandline/secret_create.md [hunk 2]
```diff
@@ -34,32 +34,32 @@ command on a manager node.
...
 ID                          NAME                    CREATED                                   UPDATED                                   SIZE
-mhv17xfe3gh6xc4rij5orpfds   secret.json             2016-10-27 23:25:43.909181089 +0000 UTC   2016-10-27 23:25:43.909181089 +0000 UTC   1679
+mhv17xfe3gh6xc4rij5orpfds   my_secret               2016-10-27 23:25:43.909181089 +0000 UTC   2016-10-27 23:25:43.909181089 +0000 UTC   1679
 ```
 
 ### Create a secret with a file
 
 ```bash
-$ docker secret create --file secret.in secret.json
+$ docker secret create -f secret.json my_secret
```
Comment (mstanleyjones) [inline docs/reference/commandline/secret_create.md]: This one is right!

**Review by mdlinville — CHANGES_REQUESTED** [review]

File: docs/reference/commandline/secret_create.md (hunk 1 above)
Comment (thaJeztah) [inline docs/reference/commandline/secret_create.md]: the `-` takes the position of the file name to indicate it's reading from stdin

**Review by thaJeztah — APPROVED** [review]
LGTM

File: docs/reference/commandline/secret_create.md (hunk 1 above)
Comment (vieux) [inline docs/reference/commandline/secret_create.md]: That's how it was merged by the maintainers, I'm just helping with doc here :)

Comment (mstanleyjones) [inline docs/reference/commandline/secret_create.md]: @thaJeztah I understand what it is doing, I just did not realize that `-f` would be a required flag even when piping from STDIN. Do other Docker commands work like that?

**Review by mdlinville — APPROVED** [review]
This is accurate for RC5 though I hope we can change it later...

**Pull request merged**